compiler/compile/OMRCompilation.cpp

```diff
@@ -2734,3 +2734,19 @@ void OMR::Compilation::invalidateAliasRegion()
       new (&self()->_aliasRegion) TR::Region(_heapMemoryRegion);
       }
    }
+
+/*
+ * \brief
+ *    This query tells whether the trees might contain rdbar/wrtbar opcodes
+ *    that are not fully supported by optimizer yet
+ *
+ * \note
+ *    This query is for temporarily disable opts not supporting
+ *    the newly added rdbar/wrtbar opcodes. Subprojects can overrite
+ *    the answer. This query should be deleted eventually if
+ *    all the optimizations support the opcodes.
+ */
+bool OMR::Compilation::incompleteOptimizerSupportForReadWriteBarriers()
```

Comment:
Thanks for the documentation. I'd like to see this moved to the header file however as the header defines the API to consuming projects. Projects consuming OMR should not have to dig through the OMR implementation to find out what a particular API is supposed to achieve.

Reply:
Moved the documentation to header file as suggested.